contributed_definitions/nyaml/NXpeak_model.yaml

```diff
@@ -0,0 +1,140 @@
+category: base
+doc: |
+  Peak fitting of a one-dimensional spectrum.
+symbols:
+  doc: |
+    The symbols used in the schema to specify e.g. dimensions of arrays.
+  n_peaks: |
+    Number of peaks.
```

Comment:
Not being used (I think).
I suggest to rather add the fit region here, and if you want to fit multiple regions, use multiple instances, or arrays of regions (this would be for a global fit then?)

Reply:
removed, see also comment below.